Bio/PDB/mmtf/DefaultParser.py

```diff
@@ -0,0 +1,156 @@
+from Bio.PDB.StructureBuilder import StructureBuilder
+
+class StructureDecoder(object):
+    """
```

Comment:
Minor, but PEP257 would put that on one line as:

```
"""Class to pass the data from mmtf-python into a BioPython data structure."""
```

See https://www.python.org/dev/peps/pep-0257/#one-line-docstrings